packages/cli/src/commands/preview.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { render } from 'ink';
+import { PreviewContainer } from '../containers/PreviewContainer';
+import { ConfigProvider } from '../context/configContext';
+
+import program = require('commander');
+
+program
+  .command('preview')
+  .option('-d, --dir [dir]', 'The root folder to search components in')
+  .option(
+    '--webpack-config [path]',
+    'Path to the webpack config file, defaults to webpack.config.js'
+  )
+  .option('--port [port]', 'Port that the preview server is going to be available in')
+  .description('starts a local preview server')
+  .action(({ port }) => {
+    render(
+      <ConfigProvider config={{ dryRun: true, previewPort: port }}>
```

Comment:
is dryRun: true intended here? i thought that was a cli option as well

Reply:
Not yet. It's an internal config for now (only really needed for preview).

Reply:
I guess it can be set through the bojagirc file, but that makes no sense at all :D